package.json

```diff
@@ -14,7 +14,7 @@
     "db:studio": "drizzle-kit studio",
     "db:cleanup": "tsx lib/db/cleanup.ts",
     "stripe:webhooks": "stripe listen --forward-to http://localhost:3000/api/stripe/webhook",
-    "test": "shortest",
+    "test": "pnpm shortest",
```

Comment:
updating test script to run shortest with pnpm. This would help the user to use the most up to date version of shortest.

Reply:
This differs from the readme that says npx? Should we be consistent?